js/assessments/countWords.js

```diff
@@ -1,66 +1,79 @@
 var countWords = require( "../stringProcessing/countWords.js" );
-var AssessmentResultCalculator = require( "../calculators/assessmentResultCalculator.js" );
 var AssessmentResult = require( "../values/AssessmentResult.js" );
+var inRange = require( "lodash/number/inRange" );
 
-var getScoringConfiguration = function( i18n ) {
-	return {
-		scoreArray: [
-			{
-				min: 300,
-				score: 9,
+var countWordsAssessment = function( paper, i18n ) {
+	var wordCount = 0;
+	var score = 0;
+	var text = "";
 
-				/* translators: %1$d expands to the number of words in the text, %2$s to the recommended minimum of words */
-				text: i18n.dgettext( "js-text-analysis", "The text contains %1$d words, this is more than the %2$d word recommended minimum." )
-			},
-			{
-				range: [ 250, 300 ],
-				score: 7,
+	if ( paper.hasText() ) {
+		wordCount = countWords( paper.getText() );
```

Comment:
Could this be simplified by just saying `var wordCount = countWords ( paper.getText() );` as variable definition?
I would assume countWords returns 0 for empty text.